two_sum.py

```diff
@@ -0,0 +1,6 @@
+class Solution:
+    def twoSum(self, nums: List[int], target: int) -> List[int]:
+         for i in range(len(nums)):
+            for j in range(i,len(nums)):
+                if nums[i] + nums[j] == obje and i!=j:

```

Comment:
There are other methods for this kind of problems (more elegant and faster…sliding window)

Reply:
Obje is not defined in this file … weird